IntegrationTests/Helpers/String+NonceValidation.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+extension String {
+    var isANonce: Bool {
```

Comment:
I wonder if `isAVaildNonce` or `isPaymentMethodNonce` or something similar would be clearer here. What are your thoughts?

Reply:
I like `isValidNonce`!